contrib/sssd.spec.in

```diff
@@ -30,6 +30,12 @@
 %global build_kcm_renewals 0
 %endif
 
+%if 0%{?fedora} >= 35 || 0%{?rhel} >= 9
```

Comment:
I think it should be 'Fedora >= 39' and shouldn't specify 'rhel' version in upstream spec-file.

Reply:
Fixed